include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -975,31 +972,29 @@ struct inplace_vector
   constexpr void assign(InputIterator first, InputIterator last)
     requires(std::constructible_from<T, std::iter_reference_t<InputIterator>> &&
              std::movable<T>)
-  {
+  BEMAN_IV_FREESTANDING_DELETE({
     clear();
     insert(begin(), first, last);
-  }
+  });
   template <details::inplace_vector::container_compatible_range<T> R>
   constexpr void assign_range(R &&rg)
     requires(std::constructible_from<T, std::ranges::range_reference_t<R>> &&
              std::movable<T>)
-  {
-    assign(std::begin(rg), std::end(rg));
-  }
+  BEMAN_IV_FREESTANDING_DELETE({ assign(std::begin(rg), std::end(rg)); });
```

Comment:
existing thing, but I think these should be std::ranges based
```suggestion
  BEMAN_IV_FREESTANDING_DELETE({ assign(std::ranges::begin(rg), std::ranges::end(rg)); });
```

Reply:
@wusatosi  How would you like to proceed with these small style/cleanup related issues?
- create a separate issue+pr for each small change?
- accumulate them and submit a single pr?
- fix them now and here, even in an unrelated pr?"

Reply:
Small collateral update is fine, we can package this in.